mashafrancis/butternut-user
# feat: create and save a user account

#### Description
This adds the functionality for creating and saving a new user to the database

#### Type of change
<pre>
+ Added nodemon for the server running
+ Added resolver file for registration
+ Added graphql schema for registration
+ Added fancy generator uid in utils folder
</pre>

#### How Has This Been Tested?
1. Run `yarn start:dev`
2. Open your browser and run the following link `http://localhost:4000/`
3. Create a mutation as follows and run to see the response to return true
```
mutation { register(email: "test@test.com", password: "test") }
```

#### Story ID
#169730272
